lib/wallet/test/integration/shelley-integration-test.hs

```diff
@@ -294,32 +294,36 @@ specWithServer testDir (tr, tracers) = aroundAll withContext
 
                 mintSeaHorseAssetsLock <- newMVar ()
 
-                let clusterDir = Tagged @"cluster" testDir
                 clusterConfigs <- lookupEnvNonEmpty "LOCAL_CLUSTER_CONFIGS"
                     <&> Tagged @"cluster-configs"
                         . fromMaybe "../local-cluster/test/data/cluster-configs"
 
+                let config = Cluster.Config
+                        { cfgStakePools = error "cfgStakePools: unused"
+                        , cfgLastHardFork = era
+                        , cfgNodeLogging = error "cfgNodeLogging: unused"
+                        , cfgClusterDir = Tagged @"cluster" testDir
+                        , cfgClusterConfigs = clusterConfigs
+                        , cfgTestnetMagic = testnetMagic
+                        }
+
                 putMVar ctx Context
                     { _cleanup = pure ()
                     , _manager = (baseUrl, manager)
                     , _walletPort = CLI.Port . fromIntegral $ portFromURL baseUrl
                     , _faucet = faucet
                     , _networkParameters = np
+                    , _networkId = Cluster.testnetMagicToNatural testnetMagic
```

Comment:
Be careful not to conflate networkMagic and networkId!

https://cips.cardano.org/cips/cip34/#existingnetworkids

<img width="846" alt="Skärmavbild 2023-10-12 kl  12 06 25" src="https://github.com/cardano-foundation/cardano-wallet/assets/304423/40a70188-9353-46b6-ba8e-8d2aae4da69b"> (note: this CIP seems to be in draft, but above should still apply)

Reply:
Also this https://github.com/input-output-hk/cardano-ledger/blob/master/eras/byron/ledger/impl/doc/network-magic.md#address-discrimination-networkmagic